README.md

```diff
@@ -1,184 +1,139 @@
-`-Dhudson.slaves.NodeProvisioner.initialDelay=0,-Dhudson.slaves.NodeProvisioner.MARGIN=50,-Dhudson.slaves.NodeProvisioner.MARGIN0=0.85`.
-
-Since you changed the configuration, wait for the a new deployment to take
-effect. Once it's done, exec into the Jenkins pod and check the output of `ps
--ef`. The three configuration options we added above should up in the `java`
-command as space-separated and not comma-separated. Refer [this
-diff](https://github.com/openshift/openshift-docs/pull/7259/files?short_path=05f80f3#diff-05f80f3ab954ce57c630417065819109)
-to ensure that values are passed properly.
-
-Now lets setup weekly scan triggering mechanism. Note the pipelines will be created by seed job, but without triggers.
-
-First, setup a ConfigMap that contains URL of openshift master as below. Note, ports if any, must be included.
-```bash
-$ oc process -p OS_MASTER_URL="https://os-master.example.com:8443" -f weekly-scan/os-master-config.yaml | oc apply -f -
-```
-
-Now, we need the Jenkins token to be made available. To do this, do the following
-
-```bash
-$ oc get sa/jenkins --template='{{range .secrets}}{{ .name }} {{end}}' # Name will be something like jenkins-token-blah
-$ oc process -p JENKINS_SECRET_NAME=jenkins-token-blah -f weekly-scan/scheduler.yaml | oc apply -f -
-```
+Once the pull request is merged, CCCP:
 
+1. Links the Dockerfile
+2. Builds the image
+3. Scans / analyzes it
+4. Pushes to [registry.centos.org](https://registry.centos.org)
+5. Notifies the developer (email)
+
+Once a project is in the [CentOS Container
+Index](https://github.com/CentOS/container-index), the CentOS Container
+Pipeline Service service automatically tracks the project's Git repo and branch
+and rebuilds it every time there is a future change.
+
+## How everything works
+
+1. **Project onboard / the main "index"**
+
+    First off, the pipeline points to an index. For the CentOS community and in
+    our example, this refers to the: [CentOS Container
+    Index](https://github.com/CentOS/container-index). 
+
+2. **Jenkins and OpenShift tracking**
+
+    Jenkins is utilized in order to track each application's Git repository as
+    well as branch for any changes. This triggers a new build on **OpenShift**
+    when a change is pushed.
+    
+    Changes to the application's repository, update to the base image or any
+    RPMs that are part of the image will trigger a new build
+
+3. **Building the image**
+
+    The container image is built within OpenShift and then pushed to an
+    internal registry. This results in an image tagged `image:test`.
+
+4. **Scan and analyze the image**
+
+    Scanning happens by running scripts in the container image to check for:
+    - `yum` updates
+    - updates for packages installed via `pip`, `npm` and `gem`
+    - capabilities of the container created from resulting container image by
+      analyzing `RUN` label in the Dockerfile
+    - verify the installed RPMs by running `rpm -Va` in the resulting container
```

Comment:
*Verify the RPM installed libraries in binaries in the resulting container.

Remove the `rpm -Va` command